cmd/legacy_main.go

```diff
@@ -53,22 +55,32 @@ const (
 // Helpers
 ////////////////////////////////////////////////////////////////////////
 
-func registerSIGINTHandler(mountPoint string) {
+func registerTerminatingSignalHandler(mountPoint string, c *cfg.Config) {
 	// Register for SIGINT.
 	signalChan := make(chan os.Signal, 1)
-	signal.Notify(signalChan, os.Interrupt)
+	signal.Notify(signalChan, unix.SIGINT)
```

Comment:
shouldn't we use syscall.SIGTERM and syscall.SIGINT instead of unix.SIGINT?

Reply:
I changed back unix.SIGINT to os.Interrupt since it helps make the code platform independent. Regarding syscall.SIGTERM, the documentation of syscall recommends using [golang.org/x/sys](https://pkg.go.dev/golang.org/x/sys) wherever possible. So, I'd prefer using unix.SIGTERM.

Reply:
Got it, thanks!
Pasting for future reference, why didn't we use syscall instead? - https://go.googlesource.com/proposal/+/refs/heads/master/design/freeze-syscall.md